scripts/fiximports

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+find . -type f -name \*.go -not -name \*_cbor_gen.go | xargs -I '{}' sed -i.bak -e '/import (/ {
+  :1
+  $!N
+  s/\n\n/\'$'\n''/
+  /)/!b1
```

Comment:
I will take your word for it.

Reply:
don't take my word, take our wayward comrade @laser who I cribbed most of this from.